### PR TITLE
initial i18n

### PR DESCRIPTION
**Pull request by albemarle**
Adding the i18n framework and two sample translations.  You can put your browser into Spanish to see them.

image

### CODE REVIEW

File: src/i18n/i18n-loader.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * correctly installed with addLocaleData, all of your translations will fall back to the default
+ * (in our case English), *even if you gave IntlProvider the correct messages object for that
+ * locale*.
+ */
+
+import { addLocaleData } from 'react-intl';
+
+import enLocale from 'react-intl/locale-data/en';
+
+import esLocale from 'react-intl/locale-data/es';
```
[the remark this answers is not in the document]
Comment (albemarle) [reply src/i18n/i18n-loader.js]: We will, yes (although the list of languages need not be hardcoded).

File: package.json [hunk 2]
```diff
@@ -9,6 +9,8 @@
   "scripts": {
     "build": "NODE_ENV=production BABEL_ENV=production webpack --config=config/webpack.prod.config.js",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
+    "i18n_extract": "BABEL_ENV=i18n babel src --quiet > /dev/null",
+    "i18n_concat": "reactifex ./src/i18n/messages/src ./src/i18n/messages/translate_me.json",
```
Comment (albemarle) [inline package.json]: ...translate_me.json will get a better name soon :-)

File: src/components/UserProfile/Bio.jsx [hunk 3]
```diff
@@ -54,7 +55,13 @@ function Bio({
...
         ),
         empty: (
-          <EmptyContent onClick={() => onEdit('bio')}>Add a short bio</EmptyContent>
+          <EmptyContent onClick={() => onEdit('bio')}>
+            <FormattedMessage
+              id="profile.bio.empty"
+              defaultMessage="Add a short bio"
+              description="instructions when the user hasn't written an About Me"
+            />
+          </EmptyContent>
```
Comment (abutterworth) [inline src/components/UserProfile/Bio.jsx]: Is FormattedMessage the way we intend to add any localized string? (Even for unformatted messages?)

**Comment by abutterworth** [comment]
LGTM! 
image

**Pull request merged**